### DISCUSSION TITLE
Fox Life India

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#718] [Q&A]
**Channel Name:** Fox Life
**Channel ID:** FoxLife.in
**Country:** India
**Language:** English
**Website:** xxx
**Stream URL (optional):** xxx
**Notes (optional):** India feed of Fox Life.

**Comment by freearhey** [comment]
According to our database this channel was closed on 13 April 2024.

https://iptv-org.github.io/channels/in/FoxLife
